src/include/pbs_error.h

```diff
@@ -286,6 +286,7 @@ extern "C" {
 #define PBSE_NODE_BUSY	15227		 /* Node is busy */
 #define PBSE_DEFAULT_PARTITION 15228	/* Default partition name is not allowed */
 #define PBSE_HISTDEPEND  15229		/* Finished job did not satisfy dependency */
+#define PBSE_NOTARRAY  15230		/* Not an array job */
```

Comment:
Do we want this or just use PBSE_BADATVAL?

Reply:
To me, PBSE_BADATVAL seems unrelated to this particular error. It's not that attribute as been given a bad value but its that the attribute isn't related to the object being created.